internal/acceptance/default_namespace_test.go

```diff
@@ -0,0 +1,34 @@
+package acceptance
+
+import (
+	"context"
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/service/settings"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccDefaultNamespaceSetting(t *testing.T) {
```

Comment:
If Acc is for account, it's not correct - this is a workspace level setting

Reply:
Acc is for Acceptance. 2 lines below you can see we are targetting a workspace.